src/wp-admin/includes/misc.php

```diff
@@ -1624,8 +1624,8 @@ function wp_check_php_version() {
 
 	$response['is_lower_than_future_minimum'] = false;
 
-	// The minimum supported PHP version will be updated to 7.4 in the future. Check if the current version is lower.
-	if ( version_compare( $version, '7.4', '<' ) ) {
+	// The minimum supported PHP version will be updated to 8.0 in the future. Check if the current version is lower.
```

Comment:
```suggestion
	// The minimum supported PHP version will be updated to at least 8.0 in the future. Check if the current version is lower.
```

Based on 8.0 being significantly lower than 8.1 and 8.2 already, I think there is a decent chance that 8.0 is never the minimum.